packages/fast-css-editor-react/src/spacing/spacing.style.ts

```diff
@@ -0,0 +1,130 @@
+import { ComponentStyles } from "@microsoft/fast-jss-manager";
+
+export interface CSSSpacingClassNameContract {
+    cssSpacing?: string;
+    cssSpacing_row?: string;
+    cssSpacing_type?: string;
+    cssSpacing_type_contentRegion?: string;
+    cssSpacing_type__margin?: string;
+    cssSpacing_type__padding?: string;
+    cssSpacing_type__margin__active?: string;
+    cssSpacing_type__padding__active?: string;
+    cssSpacing_type__margin__hover?: string;
+    cssSpacing_type__padding__hover?: string;
+    cssSpacing_input?: string;
+}
+
+function styles(): ComponentStyles<CSSSpacingClassNameContract, {}> {
```

Comment:
Why are you defining this as a function only to export the return value? Maybe just export the object?